testapps/test_pillowtop/tests/test_app_pillow.py

```diff
@@ -79,3 +81,43 @@ def _create_app(self, name):
         app.save()
         self.addCleanup(app.delete)
         return app
+
+    def refresh_elasticesarch(self, kafka_seq, couch_seq):
```

Comment:
This is cool.  I bet there's a lot of situations where people might want to use a function like this in tests.  Any interest in making this into a more generalizable util?

Reply:
Yeah, that could be nice - I could see this being a test class mixin that also takes care of teardown.  Gonna call that out of scope for now, since each index is a little different, but I'll keep that in mind for the next time I need to do this.